source/extensions/extensions_metadata.yaml

```diff
@@ -1077,6 +1077,13 @@ envoy.quic.deterministic_connection_id_generator:
   status: alpha
   type_urls:
   - envoy.extensions.quic.connection_id_generator.v3.DeterministicConnectionIdGeneratorConfig
+envoy.quic.connection_id_generator.quic_lb:
+  categories:
+  - envoy.quic.connection_id_generator
+  security_posture: unknown
+  status: alpha
+  type_urls:
+  - envoy.extensions.quic.connection_id_generator.v3.DeterministicConnectionIdGeneratorConfig
```

Comment:
Should this be `envoy.extensions.quic.connection_id_generator.quic_lb.v3.Config`?